spikeinterface/sortingcomponents/peak_pipeline.py

```diff
@@ -56,7 +56,7 @@ class WaveformExtractorNode(PipelineNode):
     """Base class for waveform extractor"""
 
     def __init__(self, recording: BaseRecording, 
-                 ms_before: float, ms_after: float, parents: Optional[list[PipelineNode]]=None, return_ouput: bool=True):
+                 ms_before: float, ms_after: float, parents: Optional[List[PipelineNode]]=None, return_ouput: bool=True):
```

Comment:
This is needed for Python3.8 (and also work for > versions). Using `list[]` directly is not supported in 3.8

Reply:
So they only stared making this available after python 3.9? Good to know.

Reply:
yeah, it was failing on 3.8 envs. Using `List` seems to be compatible to both